ipc/provider/src/manager/evm/manager.rs

```diff
@@ -596,6 +605,29 @@ impl SubnetManager for EthSubnetManager {
             .await?
             .to_string())
     }
+
+    async fn get_genesis_info(&self, subnet: &SubnetID) -> Result<SubnetGenesisInfo> {
+        let address = contract_address_from_subnet(subnet)?;
+        let contract = subnet_actor_getter_facet::SubnetActorGetterFacet::new(
+            address,
+            Arc::new(self.ipc_contract_info.provider.clone()),
+        );
+        Ok(SubnetGenesisInfo {
+            // Active validators limit set for the child subnet.
+            active_validators_limit: contract.active_validators_limit().call().await?,
```

Comment:
I think it needs some type conversion first?

Reply:
I don't think so, it returns the right type right away. I tested it and it works.